src/CVCore.cpp

```diff
@@ -0,0 +1,189 @@
+#include "CVCore.h"
+
+using namespace godot;
+
+void CVCore::_bind_methods() {
+	ClassDB::bind_static_method("CVCore", D_METHOD("add"), &CVCore::add, "mat1",
+			"mat2", "mask", "dType");
+	ClassDB::bind_static_method("CVCore", D_METHOD("subtract"), &CVCore::subtract,
+			"mat1", "mat2", "mask", "dType");
+	ClassDB::bind_static_method("CVCore", D_METHOD("max"), &CVCore::max, "mat1",
+			"mat2");
+	ClassDB::bind_static_method("CVCore", D_METHOD("min"), &CVCore::min, "mat1",
+			"mat2");
+	ClassDB::bind_static_method("CVCore", D_METHOD("absdiff"), &CVCore::absdiff,
+			"mat1", "mat2");
+	ClassDB::bind_static_method("CVCore", D_METHOD("vconcat"), &CVCore::vconcat,
+			"mat1", "mat2");
+	ClassDB::bind_static_method("CVCore", D_METHOD("hconcat"), &CVCore::hconcat,
+			"mat1", "mat2");
+	ClassDB::bind_static_method("CVCore", D_METHOD("bitwise_and"),
+			&CVCore::bitwise_and, "mat1", "mat2", "mask");
+	ClassDB::bind_static_method("CVCore", D_METHOD("bitwise_or"), &CVCore::bitwise_or,
+			"mat1", "mat2", "mask");
+	ClassDB::bind_static_method("CVCore", D_METHOD("bitwise_xor"),
+			&CVCore::bitwise_xor, "mat1", "mat2", "mask");
+	ClassDB::bind_static_method("CVCore", D_METHOD("bitwise_not"),
+			&CVCore::bitwise_not, "mat", "mask");
+	ClassDB::bind_static_method("CVCore", D_METHOD("convertFp16"),
+			&CVCore::convertFp16, "mat");
+	ClassDB::bind_static_method("CVCore", D_METHOD("exp"), &CVCore::exp, "mat");
+	ClassDB::bind_static_method("CVCore", D_METHOD("log"), &CVCore::log, "mat");
+	ClassDB::bind_static_method("CVCore", D_METHOD("sqrt"), &CVCore::sqrt, "mat");
+	ClassDB::bind_static_method("CVCore", D_METHOD("transpose"), &CVCore::transpose,
+			"mat");
+}
+
+CVCore::CVCore() {
+	UtilityFunctions::push_warning("This module should not be initialized, something is wrong");
+}
+
+CVCore::~CVCore() {
+}
+
+// Arithmetic Operations
+Ref<CVMat> CVCore::add(Ref<CVMat> mat1, Ref<CVMat> mat2, Ref<CVMat> mask, int dtype) {
+	return arithmetic_wrapper(&cv::add, mat1, mat2, mask, dtype);
+}
+
+Ref<CVMat> CVCore::subtract(Ref<CVMat> mat1, Ref<CVMat> mat2, Ref<CVMat> mask, int dtype) {
+	return arithmetic_wrapper(&cv::subtract, mat1, mat2, mask, dtype);
+}
+
+Ref<CVMat> CVCore::arithmetic_wrapper(void (*func)(cv::InputArray, cv::InputArray,
+											  cv::OutputArray, cv::InputArray, int),
```

Comment:
looks like the formatting has gone wonky here. Are you using `clang-format`? If you are using Visual Studio Code, there is a plugin you can use. I tend to use "format on save" functionality, but you can do what you like as long as commits are formatted correctly.